ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/minicart/v1/minicart/clientlib/js/templates/minicart-body.js

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ *
+ *     Copyright ${year} Adobe. All rights reserved.
+ *     This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License. You may obtain a copy
+ *     of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software distributed under
+ *     the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *     OF ANY KIND, either express or implied. See the License for the specific language
+ *     governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+'use strict';
+
+const minicartBody = `<div class="miniCart__body">
```

Comment:
I would wrap the templates into a `(function() {})()` so the constants are not defined globally in addition to the `window` object.

Reply:
👍